coins/src/adapters/bridges/index.ts

```diff
@@ -129,6 +129,7 @@ import { getCurrentUnixTimestamp } from "../../utils/date";
 import produceKafkaTopics from "../../utils/coins3/produce";
 import { chainsThatShouldNotBeLowerCased } from "../../utils/shared/constants";
 import { sendMessage } from "../../../../defi/src/utils/discord";
+import { isDistressed } from "../../utils/shared/distressedCoins";
```

Comment:
I dislike the logic duplication here. 

Can you consolidates all the writes to a single function `writeToDB()`

  - this would do all the validation and filtering
  - then write to both ddb & kafka
  
this way, say I want to write something from the ui-tool or anywhere else, I know what to call, a single function that takes care of validation and storage, and in the future, we want to migrate to some other db, we update a single function